data/seeds/06-ethnicities.js

```diff
@@ -0,0 +1,18 @@
+exports.seed = function (knex) {
+  // Deletes ALL existing entries
+  return knex('ethnicities')
+    .truncate()
+    .then(function () {
+      // Inserts seed entries
+      return knex('ethnicities').insert([
+        { name: 'American Indian or Alaska Native' },
+        { name: 'Asian' },
+        { name: 'Black or African American' },
+        { name: 'Native Hawaiian or Other pacific Lslander' },
```

Comment:
typo - "or other Pacific Islander" is what you want

Reply:
I fixed the typo.